src/cloud-element-templates/properties/CustomProperties.js

```diff
@@ -98,17 +99,20 @@ function addCustomGroup(groups, props) {
     element,
     id,
     label,
+    openByDefault,
```

Comment:
This could remove the need of `typeof` check below:

```suggestion
    openByDefault = true,
```

Reply:
Good catch, I changed it with [528dcfd](https://github.com/bpmn-io/bpmn-js-element-templates/pull/18/commits/528dcfde8c66bacd049d71df204aeaf833be0ccf).